### ISSUE TITLE
SyntaxError: unqualified exec is not allowed in function 'units_wrap' it is a nested function (has_units.py, line 304)

### ISSUE THREAD

**Issue by kihneman** [enthought/scimath#80]
I get a SyntaxError testing `scimath` for OSX and Windows with Python 2. The syntax error is:
```
SyntaxError: unqualified exec is not allowed in function 'units_wrap' it is a nested function (has_units.py, line 304)
```

Here is the full traceback:
```
======================================================================
ERROR: Failure: SyntaxError (unqualified exec is not allowed in function 'units_wrap' it is a nested function (has_units.py, line 304))
----------------------------------------------------------------------

Traceback (most recent call last):
  File ".../lib/python2.7/site-packages/nose/loader.py", line 420, in loadTestsFromName
    addr.filename, addr.module)
  File ".../lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File ".../lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File ".../lib/python2.7/site-packages/scimath/units/tests/has_units_test_case.py", line 17, in <module>
    import scimath.units.has_units as has_units_
SyntaxError: unqualified exec is not allowed in function 'units_wrap' it is a nested function (has_units.py, line 304)
```

**Comment by kihneman** [comment]
This issue can be closed because I discovered that this error only occurs on Python 2.7.6 and is resolved by using Python 2.7.13.

**Comment by mdickinson** [comment]
FWIW, here is the relevant Python issue: https://bugs.python.org/issue21591
